automan/cluster_manager.py

```diff
@@ -368,6 +377,25 @@ def update(self, rebuild=True):
                 if rebuild:
                     self._rebuild(host, home)
 
+    def delete(self, sim_dir, remotes):
+        hosts = [h.get('host') for h in self.workers]
+        if remotes[-1] == 'all':
+            remotes = hosts.copy()
+        else:
+            for remote in remotes:
+                if remote not in hosts:
+                    print('%s remote is not a worker' % (remote))
+                    sys.exit()
```

Comment:
Use an explicit exit status here, 1 for error usually.

Reply:
done